code/game/objects/items/shields.dm

```diff
@@ -237,3 +237,20 @@
 	w_class = WEIGHT_CLASS_HUGE
 	item_flags = SLOWS_WHILE_IN_HAND
 	transparent = FALSE
+
+/obj/item/shield/riot/implant
+	name = "riot tower shield"
+	desc = "A massive shield that can block a lot of attacks, can take a lot of abuse before braking."
```

Comment:
`desc = "A massive shield that can block a lot of attacks, can take a lot of abuse before braking."`
*Breaking